### PR TITLE
Many minor fixes

### PR DESCRIPTION
**Pull request by tute**
* Use `configure_environment` helper in `app_builder.rb` file
* Drop comments from environment files that get trimmed anyway
* Freeze constants in `version.rb` file
* Delete unneeded `serve_static_files_line` method
* Drop trailing whitespace in NEWS

### CODE REVIEW

**Comment by tute** [comment]
Extracted from https://github.com/thoughtbot/suspenders/pull/753

File: lib/suspenders/app_builder.rb [hunk 1]
```diff
@@ -189,10 +169,9 @@ def setup_asset_host
         "config.assets.version = '1.0'",
         'config.assets.version = (ENV["ASSETS_VERSION"] || "1.0")'
 
-      inject_into_file(
-        "config/environments/production.rb",
-        '  config.static_cache_control = "public, max-age=#{1.year.to_i}"',
-        after: serve_static_files_line
+      configure_environment(
+        "production",
+        'config.static_cache_control = "public, max-age=#{1.year.to_i}"',
```
Comment (mike-burns) [inline lib/suspenders/app_builder.rb]: How about:

```ruby
%{config.static_cache_control = "public, max-age=#{1.year.to_i}"}
```

Comment (tute) [inline lib/suspenders/app_builder.rb]: > How about:
> 
> %{config.static_cache_control = "public, max-age=#{1.year.to_i}"}

That runs the interpolation while generating the app, producing: `undefined method 'year' for 1:Fixnum (NoMethodError)`.

Comment (mike-burns) [inline lib/suspenders/app_builder.rb]: Ah snap.

**Pull request merged**